Projects/Core/Networking/Model/Sources/QR/QRValidate.swift

```diff
@@ -11,17 +11,74 @@ public typealias QRValidateModel = BaseResponseDTO<QRValidateResponseModel>
 
 // MARK: - DataClass
 public struct QRValidateResponseModel: Decodable, Equatable {
-  public let valid: Bool
+  public let id: String?
+  public let profileSummary: QRProfileSummary?
+  public let scheduleSummary: QRScheduleSummary?
+  public let updatedAt, method: String
+  public let note: String?
+  public let status: AttendanceType
+
+  public init(
+    id: String?,
+    profileSummary: QRProfileSummary?,
+    scheduleSummary: QRScheduleSummary?,
+    updatedAt: String,
+    method: String,
+    note: String?,
+    status: AttendanceType
+  ) {
+    self.id = id
+    self.profileSummary = profileSummary
+    self.scheduleSummary = scheduleSummary
+    self.updatedAt = updatedAt
+    self.method = method
+    self.note = note
+    self.status = status
+  }
+}
+
+// MARK: - ProfileSummary
+public struct QRProfileSummary: Decodable, Equatable {
+  public let id: String
   public let userID: Int
-  public let username: String
-  
+  public let name: String
+  public let role: SelectPart?
+  public let team: SelectTeam?
+  public let cohort: String?
+
   public init(
-    valid: Bool,
+    id: String,
     userID: Int,
-    username: String
+    name: String,
+    role: SelectPart?,
+    team: SelectTeam?,
+    cohort: String?,
   ) {
-    self.valid = valid
+    self.id = id
     self.userID = userID
-    self.username = username
+    self.name = name
+    self.role = role
+    self.team = team
+    self.cohort = cohort
+  }
+}
+
+// MARK: - ScheduleSummary
+public struct QRScheduleSummary: Decodable, Equatable  {
+public let id, title, description: String
+ public let startTime, endTime: String
+
```

Comment:
indent 수정 한번 부탁드림니다~!
```suggestion
public struct QRScheduleSummary: Decodable, Equatable  {
  public let id, title, description: String
  public let startTime, endTime: String

```

Reply:
넵!!!